src/app/items/containers/item-children-edit-list/item-children-edit-list.component.ts

```diff
@@ -117,17 +127,57 @@ export class ItemChildrenEditListComponent implements OnChanges {
     this.op?.toggle(event);
   }
 
+  updateChildPropagations(propagations: Partial<ItemPermPropagations>, childIdx: number): void {
```

Comment:
`updateChildPropagations` => `updateChildPermPropagations`
BTW, does it really "update" as it just emit a change?